src/components/Layout/index.tsx

```diff
@@ -18,14 +20,40 @@ const navBarContent = (title: string, refUrl: string, isActive: boolean = false)
     </Link>
   );
 };
-
+const { SUCCESS, ERROR } = ToastTypes;
+const SELF_URL = `${process.env.NEXT_PUBLIC_BASE_URL}/users/self`;
```

Comment:
Please make use of constants from here 👇 
https://github.com/Real-Dev-Squad/website-status/blob/develop/src/components/constants/url.js